cypress/support/pages/room_board/pageRoomBoards.js

```diff
@@ -626,6 +628,10 @@ class RoomBoards {
 			.should("contain", title);
 	}
 
+	doNotSeeFolderElementOnBoard() {
+		cy.get(RoomBoards.#folderElementSelector).should('not.exist');
```

Comment:
I think we can use here globalAssertion method `checkElementWithDataTestIdNotExists`